src/pages/Navigator-Page-2.md

```diff
@@ -0,0 +1,39 @@
+Navigator Page 2:
+
+
+      <Navigator leftRailText='Home sweet home' stepprogress={ false }>
+        <GeneralForm
+          formName='ExampleForm'
+          onSubmit={ () => alert('FormSubmitted!') }
+        >
+          <TextComponent tag='p' type={ 4 } semibold>
+            Great, just a few more questions.
+          </TextComponent>
+          <ToggleField
+            htmlFor=''
```

Comment:
@cisacke I think so.. even if it's not used in the implementation, it makes sense to have it